src/domain/collection/curatedEventList/CuratedEvents.tsx

```diff
@@ -0,0 +1,135 @@
+import { Button } from 'hds-react';
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+
+import LoadingSpinner from '../../../common/components/spinner/LoadingSpinner';
+import { CollectionFieldsFragment } from '../../../generated/graphql';
+import Container from '../../app/layout/Container';
+import { getEventIdsFromUrls } from '../../event/EventUtils';
+import { EventType } from '../../event/types';
+import styles from './curatedEventList.module.scss';
+import EventCards from './EventCards';
+import OnlyExpiredEvents from './OnlyExpiredEvents';
+import { usePaginatedEventsByIdsQuery } from './utils';
+
+const PAST_EVENTS_DEFAULT_SIZE = 4;
+export const PAGE_SIZE = 10;
+
+export const coursesListTestId = 'curated-courses-list';
+export const eventsListTestId = 'curated-events-list';
+
+interface Props {
+  collection: CollectionFieldsFragment;
+}
+
+const CuratedEvents: React.FC<Props> = ({ collection }) => {
+  const { t } = useTranslation();
+  const { eventIds, courseIds } = React.useMemo(
+    () => getEventIdsFromUrls(collection.curatedEvents),
+    [collection.curatedEvents]
+  );
+
+  return (
+    <Container className={styles.container}>
+      {!!eventIds.length && (
+        <CollectionEventsList
+          testId={eventsListTestId}
+          eventIds={eventIds}
+          eventType="event"
+          title={t('collection.curatedEvents.eventsTitle')}
+        />
+      )}
+      {!!courseIds.length && (
+        <CollectionEventsList
+          testId={coursesListTestId}
+          eventIds={courseIds}
+          eventType="course"
+          title={t('collection.curatedEvents.coursesTitle')}
+        />
+      )}
+    </Container>
+  );
+};
+
+const CollectionEventsList: React.FC<{
+  eventIds: string[];
+  title: string;
+  eventType: EventType;
+  testId: string;
+}> = ({ eventIds, title, eventType, testId }) => {
+  const { t } = useTranslation();
+  const [showAllPastEvents, setShowAllPastEvents] = React.useState(false);
+
+  const {
+    events,
+    pastEvents,
+    isFetchingMore,
+    loading,
+    onLoadMoreEvents,
+    hasMoreEventsToLoad,
+    eventCursorIndex,
+  } = usePaginatedEventsByIdsQuery(eventIds, eventType);
+
+  const visiblePastEvent = pastEvents.slice(
```

Comment:
`visiblePastEvents` might a bit better name

Reply:
or even `visibleExpiredEvents` now we use words `expired` and `past` as synonyms and synonyms should be avoided

Reply:
yes should only use one naming for past/expired events